jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/entity/AssetGroupStatsCollector.java

```diff
@@ -50,7 +51,7 @@ public class AssetGroupStatsCollector implements Constants{
     /**
      * Collect asset group stats.
      */
-    public List<Map<String, String>> collectAssetGroupStats() {
+    public List<Map<String, String>> collectAssetGroupStats(DatasourceData datasourceData) {
 
         
         log.info("Start Collecting asset group stats");
```

Comment:
Why we need to keep code below if token is not used anymore?

Reply:
removed